# Input and output topics
publish:
    control_command: '/carla/ego_vehicle/ackermann_cmd'
    feedback: '/lmpcc/controller_feedback'
    reset: 'reset'
    reset_carla: '/carla/ego_vehicle/initialpose'
    navigation_goal: '/carla/ego_vehicle/goal'

subscribe:
    state: '/carla/ego_vehicle/odometry'
    obstacles: '/carla/objects'
    waypoints: '/carla/ego_vehicle/waypoints'
    steering_angle: '/movebox/front_left_steer_joint' # There is no delay in steering!
    acceleration: '/accel/filtered'
    velocity_reference: 'rodrigo'
    occupancy_grid: costmap/costmap/costmap

visualization:
    reference_path: '/lmpcc/reference_path'
    reference_arrows: '/lmpcc/reference_arrows'
    planned_collision_space: '/lmpcc/collision_space'
    planned_trajectory: '/lmpcc/planned_trajectory'
    free_space: '/lmpcc/free_space'
